src/components/Board.js

```diff
@@ -0,0 +1,51 @@
+import Square from './Square';
+
+const Board = (props) => {
+  const {squares, onClick, winLine} = props;
+
+  const renderSquare = (i) => {
+    let isWinSquare;
+    if (winLine) {
+      isWinSquare = winLine.indexOf(i) !== -1 ? true : false;
+    } else {
+      isWinSquare = null;
+    }
+
+    return (
+      <Square 
+        value={squares[i]}
+        onClick={() => onClick(i)}
+        key={i}
+        isWinSquare={isWinSquare}
+      />
+    );
+  }
+
+  const renderBoadRow = (i) => {
+    return (
+      <div className="board-row" key={i}>
+        {(() => {
+          const arr = [];
+          for(let j = 0; j < 3; j++) {
```

Comment:
```suggestion
          for(const j = 0; j < 3; j++) {
```
jをループ内で再代入する必要がない場合はconstでいけます

Reply:
あ、失礼。const使おうはfor of 文の話でした。
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Statements/for...of

Reply:
承知しました。